test/integration_test.js

```diff
@@ -680,7 +687,9 @@ describe("Integration tests", function() {
 
     after(() => stopServer(server));
 
-    beforeEach(() => server.flush());
+    beforeEach(() => {
+      return server.flush().then(_ => api.createAccount("user", "pass"));
```

Comment:
```suggestion
      return server.flush().then(() => api.createAccount("user", "pass"));
```

(Same as https://github.com/Kinto/kinto-http.js/pull/304/files#r249295696)